README.md

```diff
@@ -2,3 +2,12 @@
 [![Build Status](https://travis-ci.org/marco-c/coverage-crawler.svg?branch=master)](https://travis-ci.org/marco-c/coverage-crawler)
 
 A crawler to find websites that exercise code in Firefox that is not covered by unit tests
+
+## Usage on local machine
+
+- Create and activate Python 3.6 virtual environment
+- In the virtual environment install requirements for the project running `pip install -r requirements`
+- To install testing utilities run `pip install -r test-requirements.txt`
```

Comment:
Creating a virtual env is not required, it should be the choice of the user.
Let's just say that the requirements need to be installed (`requirements.txt` if you only want to run the project, `test-requirements.txt` if you want to develop).